gradle/compile-wasm-multiplatform.gradle

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+kotlin {
+    wasmJs {
+        moduleName = project.name
+        nodejs()
+        compilations['main']?.dependencies {
+            api "org.jetbrains.kotlinx:atomicfu-wasm-js:$atomicfu_version"
+        }
+    }
+
+    sourceSets {
+        wasmJsMain {
+            kotlin {
+                srcDir 'jsWasm/src'
```

Comment:
What is the difference between `jsWasm` and `wasmJs`?

One of them requires a name refinement, probably one intended to be `jsAndWasmShared`

Reply:
Yes. This is definitely worth to do. Fixed